### DISCUSSION TITLE
Chapters

### DISCUSSION THREAD

**Discussion by jridyard** [sampotts/plyr#2762] [Ideas]
In looking through the documentation I didn't find anything that could be used for one specific use case in our AI call transcription app: Chapters. Essentially, similar to how markers work, it would take two time stamps and break up the audio/media player bar by "chapter"; Chapters that are specified with a label will then display that label and non-specified sections will just show the timestamp that the user is hovering. Youtube has something like this in their videos oftentimes. Here's an example of one: https://www.youtube.com/watch?v=WmR9IMUD_CY

**Comment by andrey-sylka** [comment]
We can make work around solution 
<img width="529" alt="VideoChapters3" src="https://github.com/sampotts/plyr/assets/156120342/627a6713-9eb1-4ff8-8c08-e763999a30cd">

**Reply by andrey-sylka** [reply]
but with this solution we have got some problems: 
1. Chapter labels on a preview thumbnails appear by default only at exactly time when we setup marker. But for the long videos it is really hard to point mouse on a marker on that time and see the chapter label.  to solve it i manually increase size of markers blocks on the progress bad and add events Listeners  to show chapter Label 
2.  With it solution we have got another problem: ` player.on("canplaythrough", (event)` i  try to use all possible events, but with any of them i cant guarantee that prveview thumbnail block is already exists in dom. so i need to wrap my code with some timeout that is not good for example for the users with slow connection.

**Reply by andrey-sylka** [reply]
@sampotts  could you check is it simple increase size of the markers that labels will be shown not only at the marker time but +- some delta. Or provide event: thumbnailsIsReady
